manifest-build-tools/pushToBintray.sh

```diff
@@ -209,7 +216,9 @@ deploy_package() {
 upload_content() {
   echo "Uploading ${PACKAGE_PATH}..."
 
-  if [ $(${CURL} --write-out %{http_code} --silent --output /dev/null -H X-Bintray-Debian-Distribution:${DISTRIBUTION} -H X-Bintray-Debian-Component:${COMPONENT} -H X-Bintray-Debian-Architecture:${ARCHITECTURE} -H X-Bintray-Override:1 -T ${PACKAGE_PATH} ${BINTRAY_API}/content/${BINTRAY_SUBJECT}/${BINTRAY_REPO}/${BINTRAY_PCK}/${BINTRAY_PCK_VERSION}/ ) -eq "201" ]; then
+  uploaded_file_name=$(basename ${PACKAGE_PATH})
+
+  if [ $(${CURL} --write-out %{http_code} --silent --output /dev/null -H X-Bintray-Debian-Distribution:${DISTRIBUTION} -H X-Bintray-Debian-Component:${COMPONENT} -H X-Bintray-Debian-Architecture:${ARCHITECTURE} -H X-Bintray-Override:1 -T ${PACKAGE_PATH} ${BINTRAY_API}/content/${BINTRAY_SUBJECT}/${BINTRAY_REPO}/${BINTRAY_PCK}/${BINTRAY_PCK_VERSION}/${TARGET_FOLDER}/${uploaded_file_name} ) -eq "201" ]; then
```

Comment:
I've proved there's no negative impact, but providing flexibility .

example: I uploaded a debian package, with ```--target_folder abc/xyz```
```
./pushToBintray.sh --target_folder abc/xyz     \
--user xxxx--api_key xxxx --subject rackhd-mirror  --repo debian  --package  on-tftp --version debug-1.2.3.4    --file_path  0install_2.7-3_amd64.deb  --distribution trusty  --architecture amd64 
```

then , in the remote bintray's Package file : 
 ```/dists/trusty/main/binary-amd64/Packages``` https://bintray.com/rackhd-mirror/debian/on-tftp#files/dists/trusty/main/binary-amd64

it will contains:
```
Package: 0install
...
Filename: abc/xyz/0install_2.7-3_amd64.deb
```

and apt-get update && apt-cache search can find this package as well.